src/surbhi/SumOfDigit.java

```diff
@@ -0,0 +1,53 @@
+package surbhi;
+public class SumOfDigit {
+	/************
+	 * Condition of program1-program 1: sum of all digits from a given string input:
+	 * te1ch9kj8 output:= 18
+	 ************************/
+
+	public int sumOfDigits(String stng) {
+		int l = stng.length();
+		int sum = 0;
+		for (int i = 0; i < l; i++) {
+			if (Character.isDigit(stng.charAt(i))) {
+				String tmp = stng.substring(i, i + 1);
```

Comment:
Why substring here surbhi?
you can use this method on character class
Character.getNumericValue(ch);